src/Xamarin.Android.Build.Tasks/Tasks/AndroidCreateDebugKey.cs

```diff
@@ -27,16 +27,31 @@ protected override CommandLineBuilder CreateCommandLine ()
 			return cmd;
 		}
 
+		bool warnExitCode = false;
+
 		protected override void LogFromStandardError (string text)
 		{
 			// Downgrade error message to warning:
 			// error: java.lang.Exception: Key pair not generated, alias <androiddebugkey> already exists
 			if (text.Contains ($"alias <{KeyAlias}> already exists")) {
 				text = text.Replace ("error:", "warning:");
+				warnExitCode = true;
```

Comment:
Is it at all plausible for `keytool` to emit more than one error message?  A scenario that comes to mind is "what if" we get the "alias already exists" error *as well as* some other error.  With this logic, we'll ignore *all* the errors so long as one of them contains `$"alias <{KeyAlias}> already exists"`.

"Safer" might be:

```csharp
int errorMessageCount;

protected override void LogFromStandardError (string text)
{
	if (text.Contains ("error:", StringComparison.OrdinalIgnoreCase))
		errorMessageCount++;
	if text.Contains ($"alias <{KeyAlias}> already exists")) {
		text = text.Replace ("error:", "warning:");
		errorMessageCount--;
	}

	base.LogFromStandardError (text);
}

protected override bool HandleTaskExecutionErrors ()
{
	if (errorMessageCount > 0) {
		Log.LogWarningFromResources ("ToolTask.ToolCommandFailed", ToolExe, ExitCode);
		return true;
	}

	return base.HandleTaskExecutionErrors ();
}
```

Reply:
I added a check such as:
```csharp
if (warnExitCode && !Log.HasLoggedErrors) {
	Log.LogWarningFromResources ("ToolTask.ToolCommandFailed", ToolExe, ExitCode);
	return true;
}
```
So if anything logs an error before we get to the exit code, it won't change it to a warning.

I think that will do the same thing without incrementing an `int`.